mne/epochs.py

```diff
@@ -508,6 +508,42 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
                            scalings=scalings, title_str=title_str,
                            show=show, block=block)
 
+    def plot_concat(self, picks=None, scalings=None, n_epochs=8,
+                    n_channels=10, title_str='#%003i', show=True, block=False):
+        """Visualize single trials as concatenated data.
+
+        Parameters
+        ----------
+        picks : array-like of int | None
+            Channels to be included. If None only good data channels are used.
+            Defaults to None
+        scalings : dict | None
+            Scale factors for the traces. If None, defaults to
+            ``dict(mag=1e-12, grad=4e-11, eeg=20e-6, eog=150e-6, ecg=5e-4,
+            emg=1e-3, ref_meg=1e-12, misc=1e-3, stim=1, resp=1, chpi=1e-4)``.
+        n_epochs : int
+            The number of epochs per view.
+        n_channels : int
+            The number of channels per view.
+        title_str : None | str
+            The string formatting to use for axes titles. If None, no titles
+            will be shown. Defaults expand to ``#001, #002, ...``.
+        show : bool
+            Whether to show the figure or not.
+        block : bool
+            Whether to halt program execution until the figure is closed.
+            Useful for rejecting bad trials on the fly by clicking on a
+            sub plot.
```

Comment:
Yo @jaeilepp this looks already pretty good! But when you look around each arg that has a default also has a comment that makes it explicit. Like "defaults to None" for picks. Do something similar for the other params.